src/multi-select/tp-multi-select-search.ts

```diff
@@ -86,17 +85,6 @@ export class TPMultiSelectSearchElement extends HTMLElement {
 		multiSelect.setAttribute( 'visible-options', matchedOptionCount.toString() );
 	}
 
-	/**
-	 * Handle click.
-	 *
-	 * @param {Event} e Click event.
-	 */
-	protected handleClick( e: Event ): void {
-		e.preventDefault();
-		e.stopPropagation();
```

Comment:
@junaidbhura As per mentioned in the description, the second approach would be to remove the stop propagation over here.

But removing this will no more open the dropdown.
Why:
That's because, it will open the dropdown, but the event will bubble up to the field component and there it will once again trigger its dropdown toggle function, which will instantly close the dropdown.

So, for this the simple solution would be that in the field sub-component, where we are attaching the event-listener, attach the event listener in the capturing phase and not in the bubbling phase, by setting the third parameter as true.
This will just ignore any bubbling events.

this.addEventListener( 'click', this.toggleOpen.bind( this ), true );